doc/common/abbreviations.md

```diff
@@ -285,6 +320,7 @@
 | SONiC | Software for Open Networking in the Cloud |
 | SR-IOV | Single Root Input Output Virtualisation |
 | SRTM | Static Root of Trust for Measurements |
+| SRV | (Traffic) client-SeRVer traffic |
```

Comment:
@pgoyal01 : Would you have a reference for this? I see how "SRV" can be that, but otherwise would not expect it and don't see it online.

Reply:
> @petorre The abbreviations SRV and FWD are used in RC1 - [2.8.7.1 Performance Measurements](https://github.com/cntt-n/CNTT/blob/master/doc/ref_cert/RC1/chapters/chapter02.md#2871-performance-measurements).

@pgoyal01 : OK, thanks.